ext/zip/config.m4

```diff
@@ -4,36 +4,21 @@ PHP_ARG_WITH([zip],
     [Include Zip read/write support])])
 
 if test "$PHP_ZIP" != "no"; then
-  PKG_CHECK_MODULES([LIBZIP], [libzip >= 0.11])
-  LIBZIP_LIBDIR=`$PKG_CONFIG --variable=libdir libzip`
+  PKG_CHECK_MODULES([LIBZIP], [libzip >= 1.0.1])
 
-  dnl Could not think of a simple way to check libzip for overwrite support
-  PHP_CHECK_LIBRARY(zip, zip_open,
-  [
-    PHP_ADD_LIBRARY_WITH_PATH(zip, $LIBZIP_LIBDIR, ZIP_SHARED_LIBADD)
-    AC_DEFINE(HAVE_LIBZIP,1,[ ])
-  ], [
-    AC_MSG_ERROR(could not find usable libzip)
-  ], [
-    -L$LIBZIP_LIBDIR
-  ])
+  PHP_EVAL_INCLINE($LIBZIP_CFLAGS)
+  PHP_EVAL_LIBLINE($LIBZIP_LIBS, ZIP_SHARED_LIBADD)
 
   PHP_CHECK_LIBRARY(zip, zip_file_set_encryption,
   [
-    PHP_ADD_LIBRARY_WITH_PATH(zip, $LIBZIP_LIBDIR, ZIP_SHARED_LIBADD)
     AC_DEFINE(HAVE_ENCRYPTION, 1, [Libzip >= 1.2.0 with encryption support])
   ], [
     AC_MSG_WARN(Libzip >= 1.2.0 needed for encryption support)
-  ], [
-    -L$LIBZIP_LIBDIR
```

Comment:
cannot be removed (or replaced by $LIBZIP_LIBS)

Reply:
@remicollet, they can be safely removed with PHP_EVAL_LIBLINE added.

Reply:
Sorry, but I stop following this one, too much noise for nothing

Reply:
@hughmcmaster I think that just the EVAL_LIBLINE will not be enough for shared builds. This should probably be replaced with `$ZIP_SHARED_LIBADD` instead.